asset_delivery.md

```diff
@@ -1,6 +1,10 @@
 # Asset Delivery
 
-## Option A - Use Cloudfront CDN and Cloudflare CDN
+## Option A - Use Cloudfront Directly
+
+Since Cloudfront offers simple SSL certs, this now is an option.
```

Comment:
Are there any restrictions? For example should we limit *cloudfront* to deliver only the assets?